src/main/java/core/basesyntax/process/DataProcess.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.process;
+
+import core.basesyntax.storage.FruitStorage;
+import core.basesyntax.storage.FruitTransaction;
+import core.basesyntax.strategy.ShopStrategy;
+import core.basesyntax.strategyhandler.StrategyHandlerImpl;
+import java.util.List;
+
+public class DataProcess {
+    private static final int INDEX_OPERATION_TYPE = 0;
+    private static final int INDEX_FRUIT_TYPE = 1;
+    private static final int INDEX_QUANTITY = 2;
+
+    public void addDataToStorage(List<String> list, FruitStorage fruitStorage) {
+        StrategyHandlerImpl cps = new StrategyHandlerImpl();
```

Comment:
1. bad naming
2. let's separate concerns / avoid tight coupling - we should parse transactions into models first, and then process these models in a separate service